Usage Console App (#4180)
SoftwareAG/c8y-docs: content/console/usage-bundle/data.md

Reviewer: <img width="390" height="237" alt="Image" src="https://github.com/user-attachments/assets/a9b2af1c-3a00-4106-a367-8969f346e519" />

This should be "Usage data" instead of "Usage Data" following the capitalization scheme of subtitles.

Author: Thank you, @tydunne .
Fixed this in https://github.com/Cumulocity-IoT/c8y-docs/pull/4180/commits/a19d77f96282ba6eaaf6e635e4a7f86dcd188811

---
weight: 20
title: Data
layout: redirect
sector:
  - platform_administration
---

> The **Data** page gives you access to the raw underlying data that powers the graphs in the **Overview** page. It is simply a table with all the data, available for use with the same filters as the visualizations.

Reviewer 2: ```suggestion
The **Data** page provides access to the raw underlying data that powers the graphs in the **Overview** page. It is simply a table with all data, available for use with the same filters as the visualizations.
```

> The **Usage Data** page provides access to the raw underlying data that powers the graphs on the **Usage Overview** page. It is a table with all of the data, available for use with the same filters as the visualizations.

Reviewer 2: ```suggestion
The **Usage data** page provides access to the raw underlying data that powers the graphs on the **Usage overview** page. It is a table with all data available for use with the same filters as the visualizations.
```

<img src="/images/console/usage/data/data.png" alt="Data">

> The data is available on the tenant level. You can use the **Configure Columns** option to add or remove metrics from the table.

Reviewer 2: ```suggestion
The data is available on the tenant level. You can use the **Configure columns** option to add or remove metrics from the table.
```

> <img src="/images/console/usage/data/configure-columns.png" alt="Configure columns dialog">

Reviewer 2: Again, the image is too large.

Author: Addressed in https://github.com/Cumulocity-IoT/c8y-docs/pull/4180/commits/d66dc3224e6a39b1b32493cdde4d6a4b3d5e26cd
